app/src/main/java/com/sun/colornotetaking/data/LabelRepository.java

```diff
@@ -0,0 +1,52 @@
+package com.sun.colornotetaking.data;
+
+import android.content.Context;
+
+import com.sun.colornotetaking.data.local.source.LabelDataSource;
+import com.sun.colornotetaking.data.local.source.LabelLocalDataSource;
+import com.sun.colornotetaking.data.model.Label;
+
+import java.util.List;
+
+public class LabelRepository implements LabelDataSource {
+
+    private static LabelRepository sLabelRepository;
+    private LabelDataSource mLabelDataSource;
+
+    public LabelRepository(LabelDataSource labelDataSource) {
+        mLabelDataSource = labelDataSource;
+    }
+
+    public static LabelRepository getInstance(Context context) {
```

Comment:
em inject sai đối tượng r nè
```suggestion
    public static LabelRepository getInstance(LabelDataSource local) {
```